ui/pages/Login/components/Login.jsx

```diff
@@ -2,53 +2,48 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import queryString from 'query-string'
 import { connect } from 'react-redux'
-import { Link } from 'react-router-dom'
-import { Divider, Message, Button, Icon } from 'semantic-ui-react'
+import { Redirect } from 'react-router-dom'
+import { Message, Button, Icon } from 'semantic-ui-react'
 
 import { getGoogleLoginEnabled } from 'redux/selectors'
-import { validators } from 'shared/components/form/ReduxFormWrapper'
 import { ANVIL_URL } from 'shared/utils/constants'
 import { login } from '../reducers'
-import UserFormLayout from './UserFormLayout'
-
-const FIELDS = [
-  { name: 'email', label: 'Email', validate: validators.required },
-  { name: 'password', label: 'Password', type: 'password', validate: validators.required },
-]
-
-const Login = ({ onSubmit, googleLoginEnabled, location }) =>
-  <UserFormLayout
-    header="Login to seqr"
-    onSubmit={onSubmit}
-    form="login"
-    fields={FIELDS}
-    content={googleLoginEnabled &&
+
+const Login = ({ location }) => {
+  const queryParams = queryString.parse(location.search)
+  const googleauthLoginUri = `/login/google-oauth2${location.search}`
+
+  if (Object.keys(queryParams).length === 0 && !location.pathname.includes('google-oauth2')) {
```

Comment:
Could we simplify this to `!queryString.parse(location.search).googleLoginFailed`? I don't think we need to handle any other cases.